ProjectVoyage/Cell/ChatCell.h

```diff
@@ -12,12 +12,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ChatCell : UITableViewCell
 
-@property (strong, nonatomic) IBOutlet UILabel *recipientLabel;
-@property (strong, nonatomic) IBOutlet UILabel *descriptionLabel;
-@property (strong, nonatomic) IBOutlet UIImageView *recipientImage;
-@property (weak, nonatomic) IBOutlet UILabel *dateChat;
```

Comment:
we want to ensure that all IBOutlets (unless they create a retain cycle) are strong

Reply:
more about retain cycles: https://www.cocoawithlove.com/2009/07/rules-to-avoid-retain-cycles.html